Standards/scs-0001-v2-sovereign-cloud-standards.md

```diff
@@ -0,0 +1,309 @@
+- A _Motivation_ section which details why this document or the thing it describes is necessary.
+- A _Standard_ section containing the actual standardization decision.
+
+In addition, the following OPTIONAL sections should be considered:
+
+- A _Terminology_ section which shortly describes terms used in the document, including possible abbreviations.
+- A _Design Considerations_ section for Standard type documents,
+  which details other choices
+  which have been considered for the specific feature
+  but were ultimately rejected.
+- An _Open Questions_ section which links to issues
+  detailing any open discussion points with respect to a document.
+  This section is RECOMMENDED during the discussion phase (pre 1.0.0)
+  as a "table of contents" of things to work on in that context.
+- A _Related Documents_ section which references related Standards
+  or Decisions, both upstream and/or other SCS documents.
+- A _Conformance Tests_ section that contains hints on how to validate
+  conformance with this spec, ideally links to conformance test cases.
+
+#### Decision Record
+
+Each Decision Record document MUST have the following sections:
+
+- An _Introduction_ providing context on the document and linking to other relevant materials.
+- A _Context_ section describing the issue relevant for motivating this Decision Record.
+- A _Decision_ section containing the actual decision that is introduced. The section should also include
+  reasoning for this decision.
+
+In addition, the following OPTIONAL sections should be considered:
+
+- A _Terminology_ section which shortly describes terms used in the document, including possible abbreviations.
+- A _Consequences_ section describing outcomes from implementing the changes described.
+- A _Related Documents_ section which references related Standards
+  or Decisions, both upstream and/or other SCS documents.
+
+## Process
+
+The lifecycle of an SCS document goes through the following phases:
+Proposal, Draft, Stable, Deprecated, and Rejected.
+
+```mermaid
+graph TD
+    A[Proposal] -->|Pull Request| B[Draft]
+    B -->|Pull Request| D[Stable]
+    B -->|Pull Request| E[Rejected]
+    D -->|Pull Request| F[Deprecated]
+```
+
+All decisions for phase transitions follow loose consensus,
+where the group which has to form the consensus depends on the `track` of the document:
+
+- IaaS: The team working on infrastructure-as-a-service topics
+- KaaS: The team working on Kubernetes-as-a-service topics
+- Ops: The team working on operations topics
+- IAM: The team working on identity and access management topics
+- Global: The entire SCS community
+
+### Proposal phase
+
+#### Proposal of a new document
+
+To propose a new SCS document, a community participant creates a pull request on GitHub
+against the [standards repository in the SovereignCloudStack organisation][scs-standards-repo].
+
+The pull request MUST add exactly one SCS document, in the `Standards` folder.
```

Comment:
Didn't we agree to place standards and decision records in separate folders?

Reply:
No, I don't think that was agreed on. But I can bring it up again today in the SIG meeting.